CHANGELOG.mdx

```diff
@@ -16,6 +16,14 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [2.8.15] - 2021-11-25 No significant changes.
+
+
+## [2.8.15] - 2021-11-25
```

Comment:
Why is there 2 entries for 2.8.15?

Reply:
my bad - I was trying to work out how to get the changelog to have the correct link to the issue, which is a rasa-x issue